src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> listOfBulldozers = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            listOfBulldozers.add(new Bulldozer());
+        }
+        return listOfBulldozers;
```

Comment:
Why do you create list of trucks with some random parameter values and list of Bulldozers and Excavators with no parameter values? Lets add some parameter values to Bulldozer and Excavator lists